### PR TITLE
[Term Entry] PHP Array Functions: array_pad() 

### LINKED ISSUE #4000: [Term Entry] PHP Array Functions: array_pad()
### Reason/inspiration (optional)

We would like a new term entry for [PHP Array Functions](https://www.codecademy.com/resources/docs/php/arrays) about the `.array_pad()` method. The entry should go in a new file in `docs/content/php/concepts/arrays/terms/array-pad/array-pad.md`

The entry should:
    - Have an introductory paragraph about what the method is and does.
    - Have a syntax section illustrating the function in a ` ```pseudo ` code block. 
    - An example showing the function in a ` ```php ` code block. 
    - A runnable codebyte example in a ` ```codebyte/php` block. 

    
Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by andersooi**
### Description

- Added new term entry for PHP's Array Functions: array_pad()
- Closes #4000 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-pad/array-pad.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_pad()`** method is used to pad an array to a specified length with a value. If the specified length is positve, the array is padded to the right with the value until the array's size equals to the specified length. Conversely, if the specified length is negative, the array is padded to the left with the value until the array's size equals to the absolute value of specified length. The array will not be padded if the size of the array is more than the absolute value of specified length.
```
Comment (dakshdeepHERE) [inline content/php/concepts/arrays/terms/array-pad/array-pad.md:16]:
```suggestion
The **`array_pad()`** method pads an array to a specified length with a value. If the specified length is positive, the array is padded to the right with the value until the array's size equals to the specified length. Conversely, if the specified length is negative, the array is padded to the left with the value until the array's size equals the absolute value of the specified length. The array will not be padded if the size of the array is more than the absolute value of the specified length.
```

File: content/php/concepts/arrays/terms/array-pad/array-pad.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+## Syntax
+
+```pseudo
+array_pad($array, $length, $value)
+```
+
+- `$array`: Specifies the input `array`.
+- `$length`: The new size of `array`.
+- `$value`: The value to be padded to `array` if its size is less than absolute value of `$length`.
```
Comment (dakshdeepHERE) [inline content/php/concepts/arrays/terms/array-pad/array-pad.md:26]:
```suggestion
- `$array`: Specifies the input `array`.
- `$length`: The new size of `array`.
- `$value`: The value to be padded to `array` if its size is less than the absolute value of `$length`.
```

File: content/php/concepts/arrays/terms/array-pad/array-pad.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+The `array_pad()` function returns the padded array according to the specified length with the intended value.
```
Comment (dakshdeepHERE) [inline content/php/concepts/arrays/terms/array-pad/array-pad.md:28]:
```suggestion
>> **Note:** The `array_pad()` function returns the padded array according to the specified length with the intended value.
```
Lets keep this as Note

File: content/php/concepts/arrays/terms/array-pad/array-pad.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  $paddedArray = array_pad($numbers, 5, "hello");
+  $paddedArray2 = array_pad($numbers, -5, "you");
+  $paddedArray3 = array_pad($numbers, 2, "nope");
+  print_r($paddedArray);
+  print_r($paddedArray2);
+  print_r($paddedArray3);
+?>
+```
+
+The example will result in the following output:
```
Comment (dakshdeepHERE) [inline content/php/concepts/arrays/terms/array-pad/array-pad.md:46]:
```suggestion
The example above will result in the following output:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side for you @andersooi 🚀

**Review by dakshdeepHERE — APPROVED** [review]
This looks good for second review 🚀

File: content/php/concepts/arrays/terms/array-pad/array-pad.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_pad()`** method pads an array to a specified length with a value. If the specified length is positive, the array is padded to the right with the value until the array's size equals to the specified length. Conversely, if the specified length is negative, the array is padded to the left with the value until the array's size equals the absolute value of the specified length. The array will not be padded if the size of the array is more than the absolute value of the specified length.
```
Comment (avdhoottt) [inline content/php/concepts/arrays/terms/array-pad/array-pad.md:16]:
```suggestion
The **`array_pad()`** method pads an array to a specified length with a value. If the specified length is positive, the array is padded to the right with the value until the array's size equals the specified length. Conversely, if the specified length is negative, the array is padded to the left with the value until the array's size equals the absolute value of the specified length. The array will not be padded if the size of the array is more than the absolute value of the specified length.
```

**Review by avdhoottt — APPROVED** [review]
Looks good to get merged!!

**Pull request merged**